local-volume/helm/provisioner/templates/provisioner-cluster-role-binding.yaml

```diff
@@ -14,6 +14,16 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: local-storage-provisioner-node-binding-clusterrole
```

Comment:
remove the "binding" from the name

Reply:
Can you also update the example specs under deployment/kubernetes/

Reply:
done